scripts/wheels/linux/__init__.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
@Hubertus248 we did some important changes here in the last days, please use the latest __init__.py from the master

Reply:
done

Reply:
I think sth went wrong, it is not the latest one:
https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_interface/__init__.py

Reply:
Yep, that was weird. Should be fine now